xero_accounting.yaml

```diff
@@ -23317,6 +23332,21 @@ components:
           readOnly: true
           type: string
           format: uuid
+        FromIsReconciled:
+          description: The Bank Transaction boolean to show if it is reconciled for the source account
+          readOnly: true
```

Comment:
@samedwards - Are the FromIsReconciled and ToIsReconciled actually readOnly? Is the intention that the user can now call the BankTransaction endpoint and mark the corresponding transactions as reconciled as a separate step?

Reply:
Good catch, will update